nginx.conf

```diff
@@ -20,11 +20,18 @@ http {
         ssl_certificate /etc/letsencrypt/live/tuna.cs.uwaterloo.ca/fullchain.pem;
         ssl_certificate_key /etc/letsencrypt/live/tuna.cs.uwaterloo.ca/privkey.pem;
 
-        location / { try_files $uri @app; }
+        location /cuizinart { try_files $uri @app; }
         location @app {
             include uwsgi_params;
             uwsgi_pass cuizinart:5000;
         }
+
+        location /caspar { try_files $uri @appc; }
+        location @appc {
+            include uwsgi_params;
+            uwsgi_pass cuizinart:5010; 
```

Comment:
Is `cuizinart` really the right thing here? The value needs to be the name of the target container, and I assume that's `caspar`? Otherwise, I wouldn't know how docker will distinguish where to send the request...